src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -0,0 +1,823 @@
+  gamma = (rho1/rho*equations.cvs1*equations.gamma1 + rho2/rho*equations.cvs2*equations.gamma2)/(rho1/rho*equations.cvs1 + rho2/rho*equations.cvs2)
+  p     = (gamma - 1) * (rho_e - 0.5 * rho * (v1^2 + v2^2))
+
+  return SVector(rho1, rho2, v1, v2, p)
+end
+
+
+# Convert conservative variables to entropy
+@inline function cons2entropy(u, equations::CompressibleEulerMulticomponentEquations2D)
+  rho1, rho2, rho_v1, rho_v2, rho_e = u
+
+  rho       = rho1 + rho2
+  v1        = rho_v1 / rho
+  v2        = rho_v2 / rho
+  v_square  = v1^2 + v2^2
+  gamma     = (rho1/rho*equations.cvs1*equations.gamma1 + rho2/rho*equations.cvs2*equations.gamma2)/(rho1/rho*equations.cvs1 + rho2/rho*equations.cvs2)
+  p         = (gamma - 1) * (rho_e - 0.5 * rho * v_square)
+  s         = log(p) - gamma*log(rho)
+  rho_p     = rho / p
+
+  # Multicomponent stuff
+  T         = (rho_e - 0.5 * rho * v_square) / (rho1 * equations.cvs1 + rho2 * equations.cvs2)
+  s1        = equations.cvs1 * log(T) - equations.rs1 * log(rho1)
+  s2        = equations.cvs2 * log(T) - equations.rs2 * log(rho2)
+
+  # Entropy variables
+  w1        = -s1 + equations.rs1 + equations.cvs1 - (v_square / (2*T)) # + e01/T (bec. e01 = 0 for compressible euler) (DONT confuse it with e1 which is e1 = cvs * T)
+  w2        = -s2 + equations.rs2 + equations.cvs2 - (v_square / (2*T)) # + e02/T (bec. e02 = 0 for compressible euler)
+  w3        = (v1)/T
+  w4        = (v2)/T
+  w5        = (-1.0)/T 
+
+  return SVector(w1, w2, w3, w4, w5)
+end
+
+
+#@inline function gamma(u, equations::CompressibleEulerMulticomponentEquations2D)
+#  rho1, rho2, rho_v1, rho_v2, rho_e = u
+#  rho    = rho1 + rho2
+#  gamma  = (rho1/rho*equations.cvs1*equations.gamma1 + rho2/rho*equations.cvs2*equations.gamma2)/(rho1/rho*equations.cvs1 + rho2/rho*equations.cvs2)
+#
+#  return gamma
+# end
+
+
+#@inline function gamma(u_ll, u_rr, equations::CompressibleEulerMulticomponentEquations2D)
+#  rho1_ll, rho2_ll, rho_v1_ll, rho_v2_ll, rho_e_ll = u_ll
+#  rho1_rr, rho2_rr, rho_v1_rr, rho_v2_rr, rho_e_rr = u_rr
+# 
+#  rho1    = 0.5 * (rho1_ll + rho1_rr)
+#  rho2    = 0.5 * (rho2_ll + rho2_rr)
+#  rho     = rho1 + rho2
+#
+#  gamma   = (rho1/rho*equations.cvs1*equations.gamma1 + rho2/rho*equations.cvs2*equations.gamma2)/(rho1/rho*equations.cvs1 + rho2/rho*equations.cvs2)
+#
+#  return gamma
+# end
+
+
+# Convert primitive to conservative variables
+@inline function prim2cons(prim, equations::CompressibleEulerMulticomponentEquations2D)
+  rho1, rho2, v1, v2, p = prim
+  rho     = rho1 + rho2
+
+  gamma  = (rho1/rho*equations.cvs1*equations.gamma1 + rho2/rho*equations.cvs2*equations.gamma2)/(rho1/rho*equations.cvs1 + rho2/rho*equations.cvs2)
+  #gamma   = gamma(prim, equations)
+  rho_v1  = rho * v1
+  rho_v2  = rho * v2
+  rho_e   = p/(gamma-1) + 0.5 * (rho_v1 * v1 + rho_v2 * v2)
+
+  return SVector(rho1, rho2, rho_v1, rho_v2, rho_e)
+end
+
+
+#@inline function density(u, equations::CompressibleEulerMulticomponentEquations2D)
```

Comment:
OK, last time I post it: Please remove all commented code, or, if you think it can be useful, just add a manual test for it (see, e.g., `test/test_manual.jl`)